### DISCUSSION TITLE
How to change PHP version?

### DISCUSSION THREAD

**Discussion by Phil-War** [leokhoa/laragon#377] [General]
Hi,
I am desperately looking for how I can change to PHP 8.x.x

Any links will be greatly appreciated.

long live laragon!

**Comment by ahm3dou** [comment]
Hello 

I think you have to download the php version from php official website and extract it in the laragon's php folder D:/laragon/bin/php in my case and then go to laragon app  Right click -> php -> version and you should see your new version in the list, select it and restart laragon

Ouala

**Reply by Phil-War** [reply]
perfect! thank you

**Comment by Pen-y-Fan** [comment]
I have written a blog post with full details: 

- [How to update the PHP version in Laragon](https://pen-y-fan.github.io/2023/01/15/how-to-update-the-php-version-in-laragon/)
